FogCreek/Glitch-Community
# Olive team: Show Collection Notes on the Homepage

Gif: http://g.recordit.co/dKfDOQgF86.gif

Changes:
- if a collection's featured projects have at least one note it will show the note and hide the description
- I combined our two `loadCollection` functions, feel free to take an extra close eye there to make sure I'm not somehow breaking something by doing that. 
- I removed the projectLoader here as I don't think we need it...but again feel free to take a close 👁 to make sure that doesn't have a bad side effect I'm not thinking about
- some flexbox-y css stuff to make sure things are vertically aligned properly when we have notes

How to test:
- go here: https://glitch.com/edit/#!/olive-team?path=src/curated/collections.js
- add the collections you want to test with. For my testing I used these:
```
  { owner: 'glitch', name: 'probot-your-bff-on-github' },
  { owner: 'glitch', name: 'make-me-multilingual' },
  { owner: 'glitch', name: 'code-with-comics' },
  { owner: 'glitch', name: 'sketch-with-p-5' },
  { owner: 'clever-assembly-test', name: 'forest-melange' },
  { owner: 'empty-team', name: 'markdown-overflow-testing' }
```
but feel free to add your own... I have noticed we don't seem to support user collections (only team collections) on the homepage at this time which is interesting, but that's a current bug on prod I think, so I decided not to tackle that yet. 
- **make sure to reset the curated collections to whatever they were before**! Going to try my very hardest to make sure I don't put a test collection on the homepage when I deploy this lol

Things left to do before deploying:
- [x] qa logged in user
- [x] qa logged out user
- [x] qa firefox
- [x] qa safari
- [x] qa edge
